README.md

```diff
@@ -62,12 +62,76 @@ Use `action-validator -h` to see additional options.
 > repo -- or, heaven forfend, outside the repository entirely.
 
 
+## Pre-commit hook example
+
+Create an executable file in the .git/hooks directory of the target repository:
+`touch .git/hooks/pre-commit && chmod +x .git/hooks/pre-commit` and paste the following example code:
+
+```bash
+#!/bin/bash
+if [ -d ".github/workflows" ]; then
+  if command -v action-validator >/dev/null 2>&1; then
+    echo "Running pre-commit hook for GitHub Actions: https://github.com/mpalmer/action-validator"
+    scan_count=0
+    for action in $(find .github/workflows -name "*.y*ml"); do
+      validate="$(action-validator "$action")"
```

Comment:
Use the exit status of the command rather than examining the output; there's no guarantee that it'll always be silent on success.  Good work quoting the filename, though; so many people miss that in shell scripts.

Reply:
I tried this first, but the pre-commit hook was not blocking the commit when there is an error.

```
...[error]
Fatal error validating .github/workflows/broken.yaml: validation failed
action-validator scanned 1 GitHub Actions found no errors!
[main e279f91] update read-me
 2 files changed, 2 insertions(+), 2 deletions(-)
```
As for the for loop... I thought of a better way to do this. stand by for next commit

Reply:
What you need to do here is capture the exit code of `action-validator` into a variable (`rc="$?"`) and then test whether that is zero or non-zero; that'll tell you whether or not `action-validator` accepted or rejected the YAML file.

Reply:
Gotcha. I did get a shellcheck linting error for this syntax though. Instead, the recommended syntax is to just test the original variable itself. https://github-wiki-see.page/m/koalaman/shellcheck/wiki/SC2181

Take a look at my newest commits to see if this looks better to you.